routing/router.go

```diff
@@ -338,3 +343,36 @@ func serveIndexFile(w http.ResponseWriter) {
 		log.Errorf("HTTP I/O error [%v]", err.Error())
 	}
 }
+
+// Things to help build the logger handler chain (see https://pkg.go.dev/github.com/rs/zerolog/hlog#example-package-Handler)
+// fake alice to avoid dep
+type alice struct {
+	m []func(http.Handler) http.Handler
+}
+
+func (a alice) append(m func(http.Handler) http.Handler) alice {
+	a.m = append(a.m, m)
+	return a
+}
+func (a alice) then(h http.Handler) http.Handler {
+	for i := range a.m {
+		h = a.m[len(a.m)-1-i](h)
+	}
+	return h
+}
```

Comment:
Why not just import the dep?

Reply:
I followed the example that does this: https://pkg.go.dev/github.com/rs/zerolog/hlog#example-package-Handler

We don't need yet another dep to manage just for this - all we need are those two simple functions to do what we need here.